src/openklant/components/klantinteracties/models/mixins.py

```diff
@@ -42,14 +44,13 @@ class BezoekadresMixin(models.Model):
     bezoekadres_land = models.CharField(
         _("land"),
         help_text=_(
-            "Een code, opgenomen in Tabel 34, Landentabel, die het land (buiten Nederland) "
-            "aangeeft alwaar de ingeschrevene verblijft."
+            "ISO 3166-code die het land (buiten Nederland) aangeeft waar de abonnee woont."
```

Comment:
Not sure if the second part was changed on purpose?
```suggestion
            "ISO 3166-code die het land (buiten Nederland) aangeeft alwaar de ingeschrevene verblijft."
```